js/models/languagesMd.js

```diff
@@ -6407,10 +6541,21 @@ module.exports = Backbone.Model.extend({
           retry: "Retry connection", //notTranslated 
         }       
       },
+      
       {
+        
         langName: "Polski",
         langCode: "pl",
-        /* Use capitalized keys for widely reused text that must be capitalized */
+        
+        /*
+        
+        Translator: @Mido
+        02/18/2016: ~ 71 untranslated strings
```

Comment:
I like this addition, but I just want to make sure that we all are able to keep this data in sync as it's not automated at present (how many strings untranslated that is).